resolwe_bio/processes/import_data/tabular.yml

```diff
@@ -0,0 +1,55 @@
+# ===================
+# Import tabular file
+# ===================
+---
+
+- slug: upload-tab-file
+  name: Tabular file
+  data_name: '{{ src.file|default:"?" }}'
+  version: 0.0.1
+  type: data:file:tab
+  category: upload
+  persistence: RAW
+  description: >
+    Upload tabular file.
+  input:
+    - name: src
+      label: CSV file
+      type: basic:file
+      description: >
+        Tabular file. Supported extensions: .csv, .csv.gz (preferred) or .csv.*
+      validate_regex: '(\.(csv|tab|tsv|txt|xls|xlsx)(|\.gz|\.bz2|\.tgz|\.tar\.gz|\.tar\.bz2|\.zip|\.rar|\.7z))|(\.bz2)$'
+  output:
+    - name: file
+      label: Tabular file
+      type: basic:file
+    - name: src_file
+      label: Source file
+      type: basic:file
+      required: false
+  run:
+    runtime: polyglot
+    bash: |
+      if [[ '.{{ src.file }}' =~ \.(xls)$ ]]; then
```

Comment:
Cant't you just do this instead of all the code here in `bash`?
```
mv "{{ src.file_temp|default:src.file }}" "{{ src.file }}"
FILENAME=$(basename "{{ src.file }}")  # get file name
NAME="${FILENAME%.*}"  # remove extension
parse_tabular_file.py "{{ src.file }}" "${NAME}.tab.gz"
```